examples/probe-modules/module_tcp_cisco_backdoor.c

```diff
@@ -64,7 +65,7 @@ static int synscan_init_perthread(void *buf, macaddr_t *src, macaddr_t *gw,
 
 static int synscan_make_packet(void *buf, UNUSED size_t *buf_len,
 			       ipaddr_n_t src_ip, ipaddr_n_t dst_ip, uint8_t ttl,
-			       uint32_t *validation, int probe_num,
+			       uint32_t *validation, int probe_num, aesrand_t *aes,
```

Comment:
I think we need to make sure here that we're not passing in the same entropy that's passed in from the `uint32_t *validation` parameter. It might make sense to pass in IPID directly too since that's more tightly typed.

Reply:
Fair, I've moved to only generating the `ip_id` in `send.c` [here](https://github.com/zmap/zmap/pull/798/files#diff-33c4faff4dd3631a679cba7327bb63d614db66b2d3d29899ecb9bf30462a9085R233) and passing it directly in.

Also, I changed to using `random_bytes` which reads from `/dev/urandom`, should be completely distinct from any other RNG in our code.